### ISSUE TITLE
Trying to auth to vault with certs

### ISSUE THREAD

**Issue by arkpoah** [criteo/consul-templaterb#67]
Hello,

I'm not able to connect to vault with certs.
My consul-templaterb version is 1.27.0

I have a `vault_test.erb` file with : 
```
<%= secret("ssl/cloudflare/origin/key")['data'] %>
```

I run this command : 
```
consul-templaterb --vault-addr http://vault_address:8200 --vault-cert-chain /etc/consul-templaterb/ssl/vault_auth.crt --vault-private-key /etc/consul-templaterb/ssl/vault_auth.key -o -t vault_test.erb:vault_test.txt
```

I get this error : 
```[FATAL][TEMPLATE EVALUATION ERROR] vault_test.erb:1:in `render': You need to provide a vault token to use 'secret' keyword```

I succeed to connect to vault via vault cli with cert method.

Authentification without vault token is not possible yet ?

Thanks

**Comment by pierresouchay** [comment]
Hello @arkpoah ,

Yes, you need `--vault-token` to be specified (or alternatively, you can use `VAULT_TOKEN` environment variable)

**Comment by arkpoah** [comment]
ok thx
Is it planned to be able to connect via cert method only ?

**Comment by pierresouchay** [comment]
@arkpoah No, but we accept Pull Requests :) I would be glad to add support for it
